canopy/design-system/src/styles/layout.scss

```diff
@@ -60,3 +60,79 @@ $sizes: (
     padding-left: $size;
   }
 }
+
+@for $i from 0 through 5 {
+  .border-#{$i} {
+    border: ($i * 1px) solid;
```

Comment:
should be broken out instead of using convenience syntax:

border-style and border-width

Consider a mixin.

Reply:
Let me know if the new version is what you had in mind.